tests/test_api_bugs.py

```diff
@@ -10,7 +10,7 @@ def test_missing_title(self):
         """
         Papers with the title "0" do not have a title element in the Atom feed.
 
-        It's unclear whether other falsey titles (e.g. "False", "null", or empty
+        It's unclear whether other false titles (e.g. "False", "null", or empty
```

Comment:
"Falsey" (alternatively, "falsy") is intentional here: https://developer.mozilla.org/en-US/docs/Glossary/Falsy

```suggestion
        It's unclear whether other falsey titles (e.g. "False", "null", or empty
```

Reply:
Oh sorry, my bad. I'll commit your suggestion then.